### DISCUSSION TITLE
Problema al compilar

### DISCUSSION THREAD

**Discussion by FradexMF** [IIC2133-PUC/2024-1#162] [Tarea 2]
Hola, al intentar compilar con el makefile me produce este error. 
![image](https://github.com/IIC2133-PUC/2024-1/assets/130924971/de5838ed-50c8-48a3-8234-77b0d81593a4)

Mientras trabajaba en la parte 1 hice las carpetas manualmente y ahí me dejo de causar error, pero vi que tenia error de compilar en el podio así que descargue una copia de mi repo y el error persiste. Supongo que esto significa que al corregir mi tarea produciría el mismo error.

¿Cuál podría ser una razón para esto? no cambie nada a como nos entregaron el repo y el único cambio que hice en el makefile fue el de la optimización (tenia el mismo error antes de cambiarla).

Gracias de antemano!

**Comment by tadonisb** [comment]
Tuviste alguna respuesta? Porque llevo con este problema hace *mucho* rato sin solución, incluso revisando el Makefile no tiene sentido que este error salga. 

En las reglas del Makefile hay una que directamente indica ´´´# Esta regla crea los directorios donde se guardan los archivos de objeto .o´´´, que por alguna razon no se esta ejecutando.

:/

**Reply by FradexMF** [reply]
aun no, sigo esperando

**Reply by sebacarrascop** [reply]
hola!, es algo extraño, podrías dar más detalles para intentar replicar/conocer más el error. 
Que sistema operativo usas, que comandos estás haciendo para ejecutar la tarea, y todo lo que consideres relevante.

**Reply by FradexMF** [reply]
Estoy en windows 10, esta es la terminal que estoy usando
![image](https://github.com/IIC2133-PUC/2024-1/assets/130924971/36671cb5-a040-4e29-a402-e4938a316f8b)

ahora recuerdo que añadi la opcion de hacer shift + rightclick en una carpeta para tener la opcion de abrir la terminal de linux en esa raiz, lo saque de este link https://www.tenforums.com/tutorials/179549-add-open-windows-terminal-expandable-context-menu-windows-10-a.html

el comando que uso es make nombre , desde la carpeta raiz del repositorio
